iscsi/initiator.go

```diff
@@ -3,12 +3,14 @@ package iscsi
 import (
 	"bufio"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
It should be 
```
...
"time"

"github.com/pkg/errors"
"github.com/sirupsen/logrus"

"github.com/longhorn/go-iscsi-helper/util"
```

Reply:
I am sorry. I missed this again. But why do we put `github.com/longhorn/go-iscsi-helper/util` to a separate section on the bottom?

Reply:
Because it's the current package, so it's first party, not third party.

Reply:
Thank you